test/cli_create_test.go

```diff
@@ -305,7 +305,7 @@ func (suite *PouchCreateSuite) TestCreateWithEnv(c *check.C) {
 	env3 := "TEST3"     // should not in container's real env
 	env4 := "TEST4=a b" // valid
 	env5 := "TEST5=a=b" // valid
-	res := command.PouchRun("run", "-d",
```

Comment:
why change to create here?

Reply:
I think this is "create" test, So I feel "create+start" is more suitable than "run".